pywt/_extensions/meson.build

```diff
@@ -75,12 +75,12 @@ if have_c99_complex
   c_args += ['-DHAVE_C99_COMPLEX', '-DCYTHON_CCOMPLEX=1']
 endif
 
-libc_wt = static_library('c_wt',
-  sources,
-  c_args : c_args,
-  include_directories : 'c',
-  dependencies : py_dep,
-)
```

Comment:
Was this change helpful for debugging?

Reply:
I didn't know if it was required or not, since this was the way that Meson uses to declare Cython extensions. However, I don't know if it needs to be restored?  I deleted it in order to gain more control over the flags passed to Cython

Reply:
This is a pure C library that was built once and then linked into every Cython-generated extension module. The setup with `py_dep` was admittedly a little weird though; I think it's only used to retrieve `Py_ssize_t` from `Python.h`, nothing else. Cython isn't involved here.

I have a slight preference for restoring this the way it was, to not have to double check in detail whether the change changed behavior or results in double compilation of `.c` files in `sources`.